dist/routable-component.js

```diff
@@ -261,21 +261,21 @@ var routableComponent =
 	        }
 	      }
 	      delete this.components[name];
-	      this._listeners[name];
+	      delete this._listeners[name];
 	    }
 	  }, {
-	    key: '_attach_route_event',
-	    value: function _attach_route_event(route) {
+	    key: '_attachRouteEvent',
+	    value: function _attachRouteEvent(route) {
 	      var _this2 = this;
 	
 	      var listener = function listener() {
 	        var _controllers$route$co;
 	
 	        if (!_this2.controllers[route.controller]) {
-	          if (!(route.controller in _this2.controller_classes)) {
+	          if (!(route.controller in _this2.controllerClasses)) {
 	            throw new Error('controller [' + route.controller + '] not found');
 	          }
-	          _this2.controllers[route.controller] = new _this2.controller_classes[route.controller](_this2);
+	          _this2.controllers[route.controller] = new _this2.controllerClasses[route.controller](_this2);
```

Comment:
__[eslint]__
* Error - Line 278 exceeds the maximum line length of 100. (max-len)
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)

<!-- Comment by [SideCI](https://sideci.com) -->